styles.js

```diff
@@ -7,9 +7,55 @@ export default createGlobalStyle`
     box-sizing: border-box;
   }
 
-  body {
-    margin: 0;
-    
-    font-family: system-ui;
-  }
+  //this code resets all standard browser settings
```

Comment:
having a css reset is better than having none, but this one appears to be from 2011 and a lot happened in the meantime.
I would advise taking a look at [this css reset](https://www.joshwcomeau.com/css/custom-css-reset/).

There is a lot less happening and the author (Josh Comeau) explains his decisions.

Reply:
hey Niklas , thanks for this advice i will take a look at this , but for now i am ok with that because i tested it on a lot of devices and the appearing of the design is consistent and i would have to change a lot when using this one, i think!

Reply:
thanks for your advice .but i think for now i will leave it like it is because everything works fine on different devices and browsers. maybe i will take a look at this later because i would have to rework a lot for example my headings and their  sizes and positions

Reply:
yes, that's a fair decision. You might be doing some refactoring at some point and that can be a good time to take another look at the link I provided 👍